pkg/proc/proc.go

```diff
@@ -88,8 +88,17 @@ func Continue(dbp Process) error {
 		return &ProcessExitedError{Pid: dbp.Pid()}
 	}
 	dbp.CheckAndClearManualStopRequest()
```

Comment:
I'm trying to remember the rationale behind calling this directly before the loop, is this simply to clear state before we enter the loop? 

Is there any way we can avoid this in favor of simply calling this at the top of the loop and not directly before entering.

Reply:
Yes, just to clear the flag in case the manual stop request was received while the process was stopped. We could use a flag to detect that we are on the first iteration of the loop, IMHO it's clearer this way.